README.md

```diff
@@ -53,5 +53,4 @@ def output_data() -> dict:
 
 
 ## Logger
-The logger object is singleton that is preconfigured to leverage the [rich library](https://github.com/Textualize/rich) and includes a custom "success" level.
-
+The logger is a simple module that provides wraps some rich console commands for consistency purposes.
```

Comment:
I liked the link to the rich library to make it clear what kind of "richness" the code has. What do you think about leaving the link in?

Also, I think there's a typo in `provides wraps`.

Reply:
Whoops! Should be all set now (852f9be52b76005a5fb4dda700fbd262a84147f2)!

Reply:
LGTM. Thanks.